test/e2e/case5_policy_automation_test.go

```diff
@@ -140,6 +140,17 @@ var _ = Describe("Test policy automation", func() {
 
 				return len(ansiblejobList.Items)
 			}, 30, 1).Should(Equal(1))
+			By("Job TTL should match default (1 day)")
+			Eventually(func() interface{} {
+				ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
+					context.TODO(), metav1.ListOptions{},
+				)
+				Expect(err).To(BeNil())
+				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
+				Expect(err).Should(BeNil())
+
+				return ansiblejobList.Items[0].Object["spec"].(map[string]interface{})["job_ttl"]
```

Comment:
Could you add an assertion that the length is what you expected (i.e. not 0)?

Reply:
That assertion should be checked by the `Consistently()` block above it right?

Reply:
I missed that. I think that should be fine.